base/database/testing.go

```diff
@@ -158,6 +158,16 @@ func CheckAdvisoriesAccountData(t *testing.T, rhAccountID int, advisoryIDs []int
 	assert.Equal(t, systemsAffected*len(advisoryIDs), sum, "sum of systems_affected does not match")
 }
 
+func CheckSystemUpdatesCount(t *testing.T, systemID int, check func(count []int)) {
```

Comment:
Change it to `CheckSystemUpdatesGreaterThen(t, systemID, nExpectedPackages, nUpdatesGt)`. You don't need to use func param. The first test will have zero updates greater then 1. The second one will have some number of them.

Reply:
I've rather modified the function to return the actual list, and done the checking in test methods.